pkg/server/server_controller.go

```diff
@@ -252,6 +252,21 @@ func (c *serverController) httpMux(w http.ResponseWriter, r *http.Request) {
 	s, err := c.getOrCreateServer(ctx, tenantName)
 	if err != nil {
 		log.Warningf(ctx, "unable to start server for tenant %q: %v", tenantName, err)
+		// Clear session and tenant cookies after all logouts have completed.
```

Comment:
ok to trash TenantSelectCookieName

however I think we should keep the _remaining_ tenants in MultitenantSessionCookieName.

Reply:
I'm not so sure this is helpful because then I put the UI in a state where there's no `tenant` cookie but a list of sessions. Do we expect the UI to pick one randomly? Should we pick a random tenant to assign the `tenant` cookie to instead?

My take is that a missing tenant tells me that "enough has changed" about the backing DB that we should logout completely.

Reply:
If the multitenancy session has some part that is still valid, the console should still be able to see it. So I'd suggest making the tenant cookie point to that remaining tenant.

Consider the following scenario:

1. Multilogin to tenant A and B
2. Tenant B is dropped
3. ??? I think tenant A should still be available for use in console.

Also, this reminds me that we should also support this use case:

1. Login to tenant A
2. Another tenant B is created
3. Login to tenant B
4. Switch between A and B becomes possible.

I don't believe this is currently possible but i think we can make it happen. But this may be a separate issue. Should i file it?